prow/external-plugins/cherrypicker/server.go

```diff
@@ -427,7 +427,7 @@ func (s *Server) handle(l *logrus.Entry, requestor string, comment *github.Issue
 		push = s.push
 	}
 	// Push the new branch in the bot's fork.
-	if err := push(repo, newBranch); err != nil {
+	if err := push(newBranch); err != nil {
```

Comment:
This was the only usage of `push` and as you can see on 361 it's the identical repo as was used in the original `Clone` call